practice-app/back-end/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "node -r esm src/index.js",
     "dev": "nodemon -r esm src/index.js",
-    "test": "mocha -r esm --timeout 100000 test/*.test.js test/**/*.test.js test/*.js",
+    "test": "mocha -r esm --timeout 100000 test/test_scholars.js",
```

Comment:
@alifurkanbudak This is a temporary change for development I suppose, can you revert it?

Reply:
I reverted those changes.